ifrs17/Test/ImportStorageTest.ipynb

```diff
@@ -590,6 +932,195 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Openings 1",
```

Comment:
I would start from some easy tests like. 
- Reimport of Opening BE
- Import of Opening SCENARIO (having BE in the DB)
- Import of a CF Scenario1 (having (ifrsVar incl Opening) BE and Scenario2 in the DB)
- Import of a CF Scenario1 (having (ifrsVar incl Opening) BE and Scenario1 in the DB)

Reply:
These tests have been implemented in the ImportStorageTest in the following sections: 
- 3.6
- 4.1-5
- 4.6
- 4.7

Reply:
However in this test notebook we miss a method to check CF import. Therefore 4.6 and 4.7 are only drafted. What shall we do?

Reply:
We can extend the method

Reply:
ok done